tck/running_the_tck.asciidoc

```diff
@@ -109,3 +109,10 @@ Debugging is dependent on the TCK implementor. The implementor configures Arquil
 to deploy the test deployments and run the LRA client side implementation there. In general we can say that
 the runtime needs to define java debug properties `-Xdebug -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=y`.
 Then debugger may be connected to port 8787 and to track the test execution.
+
+=== Additional notes
+
+* `TckTestBase#replayEndPhase(String)` method, used for `@Status` and `@Forget` calls 
```

Comment:
> used for `@Status` and `@Forget` call verifications, expects that recovery happens on all active LRAs in a reasonably time frame

-> adjusted a few words.

Aren't we using the (configurable) timeouts also for these calls? Then we should refer to the parameter to change the factor for the timeout here also.

Reply:
Is this discussion resolved?

Reply:
@rdebusscher the updated text says:

> ...a reasonable time frame defined by the configuration property `lra.tck.recovery.timeout`.

Reply:
ie I think this PR is in a state where it can be committed - please could you add your approval?